### ISSUE TITLE
Checks 7.81 and 7.82 incorrectly return es domain names

### ISSUE THREAD

**Issue by Jae-Danner** [toniblyx/prowler#989]
### Error messages from checks 7.81 and 7.82

7.81 [extra781] Check if Amazon Elasticsearch Service (ES) domains has encryption at-rest enabled - es [Medium]
       INFO! us-east-1: No Amazon ES domain found
       PASS! us-east-2: Amazon ES domain hermes-test-forward-opv1 has encryption at-rest enabled

An error occurred (ValidationException) when calling the DescribeElasticsearchDomain operation: 1 validation error detected: Value 'OpenSearch' at 'domainName' failed to satisfy constraint: Member must satisfy regular expression pattern: [a-z][a-z0-9\-]+
       FAIL! us-east-2: Amazon ES domain OpenSearch does not have encryption at-rest enabled

7.82 [extra782] Check if Amazon Elasticsearch Service (ES) domains has node-to-node encryption enabled - es [Medium]
       INFO! us-east-1: No Amazon ES domain found
       PASS! us-east-2: Amazon ES domain hermes-test-forward-opv1 has node-to-node encryption enabled

An error occurred (ValidationException) when calling the DescribeElasticsearchDomain operation: 1 validation error detected: Value 'OpenSear
ch' at 'domainName' failed to satisfy constraint: Member must satisfy regular expression pattern: [a-z][a-z0-9\-]+
       FAIL! us-east-2: Amazon ES domain OpenSearch does not have node-to-node encryption enabled
       INFO! us-west-1: No Amazon ES domain found
    
Code Snippet https://github.com/toniblyx/prowler/blob/master/checks/check_extra782:       
```
 extra782(){
  for regx in $REGIONS; do
    LIST_OF_DOMAINS=$($AWSCLI es list-domain-names $PROFILE_OPT --region $regx --query DomainNames --output text) 
```
    
       
**AWS CLI list-domain-name output run against our account:**
1. run list-domain-names with output returned in json format
```
$ aws es list-domain-names  --region us-east-2 **--output json**
{
    "DomainNames": [
        {
            "DomainName": "hermes-test-forward-opv1",
            "EngineType": "OpenSearch"
        }
    ]
}

```
list-domain-names output generated using query parameter in code snippet
```
$ aws es list-domain-names --region us-east-2 --query **DomainNames** --output text
hermes-test-forward-opv1        OpenSearch
```

it appears that the value of keypair "EngineType":"OpenSearch" is being returned as a domain name

I believe that the correct jmespath query parameter should be DomainNames[*].DomainName which returns the DomainName value of all DomainName objects. 
```
$ aws es list-domain-names --region us-east-2 **--query DomainNames[*].DomainName** --output text
hermes-test-forward-opv1

```

**Comment by toniblyx** [comment]
Ok, that is the change in API after name change... we have to fix probably few others related to `OpenSearch`. Thanks!

**Comment by lazize** [comment]
I believe this issue was fixed by PR #1032

**Comment by toniblyx** [comment]
that is correct, thanks! Closing it.
